compiler/ballerina-compiler-api/build.gradle

```diff
@@ -27,6 +27,11 @@ dependencies {
 
     testCompile 'org.testng:testng'
     testCompile project(path: ':ballerina-test-utils', configuration: 'shadow')
+    testCompile project(':ballerina-io')
```

Comment:
Can we test without `ballerina/io`? Just wondering if we have the requirement for this to be independent of stdlib modules.

Reply:
This is just for the tests. That should be fine right?

Reply:
Even that we usually try to avoid, right? Wouldn't it be an issue with the stdlib migration anyway?

Reply:
Yeah, had a chat with Supun as well. Will replace this with a balo test.